src/shadertools/lib/platform-defines.js

```diff
@@ -37,7 +37,7 @@ export function getPlatformShaderDefines(gl) {
     platformDefines += `\
 #define INTEL_GPU
 #define INTEL_FP64_WORKAROUND 1
-#define NVIDIA_EQUATION_WORKAROUND 1\n \
```

Comment:
@ibgreen @shaojingli , not sure why we have this extra `\n \` at end of the line, could be causing any issues? removed.